packages/formstr-app/src/containers/CreateFormNew/components/QuestionCard/UploadImage.tsx

```diff
@@ -112,15 +101,48 @@ const UploadImage: React.FC<Props> = ({ onImageUpload }) => {
           <Input.TextArea
             placeholder="Enter image URL"
             value={urlInput}
-            onChange={(e) => setUrlInput(e.target.value)}
+            onChange={handleUrlInputChange}
             style={{ marginBottom: 16 }}
             rows={4}
             aria-label="Image URL input"
           />
+          {urlInput && (
+            <div style={{ marginBottom: 16 }}>
+              {previewError ? (
+                <Alert
+                  message="Invalid image URL"
+                  type="error"
+                  showIcon
+                  style={{ marginBottom: 16 }}
+                />
+              ) : (
+                <div className="image-preview-container" style={{
+                  border: '1px solid #d9d9d9',
+                  borderRadius: '8px',
+                  padding: '8px',
+                  marginBottom: '16px'
+                }}>
+                  <p style={{ margin: '0 0 8px 0', color: '#666' }}>Preview:</p>
```

Comment:
can we move these color variables to themes, so that they can be reused

Reply:
this was a very basic style, replaced it to existing style component